gvr-x3d-demo/app/src/main/java/org/gearvrf/x3ddemo/X3DparserScript.java

```diff
@@ -70,21 +55,13 @@
  *
  */
 
-public class X3DparserScript extends GVRScript
+public class X3DparserScript extends GVRMain
```

Comment:
You might want to rename this call to X3DparserMain since it is no longer a script.

You could do it in a future pull request.